exercises/complex-numbers/complex_numbers_test.py

```diff
@@ -126,13 +115,37 @@ def test_conjugate_a_purely_imaginary_number(self):
         self.assertEqual(expected.imaginary,
                          input_number.conjugate().imaginary)
 
-    def conjugate_a_number_with_real_and_imaginary_part(self):
+    def test_conjugate_a_number_with_real_and_imaginary_part(self):
         input_number = ComplexNumber(1, 1)
         expected = ComplexNumber(1, -1)
         self.assertEqual(expected.real, input_number.conjugate().real)
         self.assertEqual(expected.imaginary,
                          input_number.conjugate().imaginary)
 
+    def test_real_part_of_a_purely_real_number(self):
+        input_number = ComplexNumber(1, 0)
+        self.assertEqual(input_number.real, 1)
+
+    def test_real_part_of_a_purely_imaginary_number(self):
+        input_number = ComplexNumber(0, 1)
+        self.assertEqual(input_number.real, 0)
+
+    def test_real_part_of_a_number_with_real_and_imaginary_part(self):
+        input_number = ComplexNumber(1, 2)
+        self.assertEqual(input_number.real, 1)
+
+    def test_imaginary_part_of_a_purely_real_number(self):
+        input_number = ComplexNumber(1, 0)
+        self.assertEqual(input_number.imaginary, 0)
+
+    def test_imaginary_part_of_a_purely_imaginary_number(self):
+        input_number = ComplexNumber(0, 1)
+        self.assertEqual(input_number.imaginary, 1)
+
+    def test_maginary_part_of_a_number_with_real_and_imaginary_part(self):
```

Comment:
Spelling error that wasn't caught in the previous version of this file. Should be
```Python
def test_imaginary_part_of_a_number_with_real_and_imaginary_part(self):
```